src/CollAction/Models/Project.cs

```diff
@@ -67,11 +67,32 @@ public bool IsClosed
 
         public List<ProjectTag> Tags { get; set; }
         public List<ProjectParticipant> Participants { get; set; }
-
+        
         [NotMapped]
         public string HashTags
             => string.Join(";", Tags?.Select(tag => tag.Tag.Name) ?? Enumerable.Empty<string>());
 
+        [NotMapped]
+        public int RemainingDays
+            => Convert.ToInt32(Math.Round((End - Start).TotalDays));
+
+        [NotMapped]
+        [Display(Name = "Status")]
+        public string StatusDescription
```

Comment:
That makes a lot of sense, I've done another commit with this moved to the DisplayProjectViewModel.